source/packages/services/installer/src/commands/modules/service/certificateVendor.ts

```diff
@@ -10,14 +10,18 @@
  *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions    *
  *  and limitations under the License.                                                                                *
  *********************************************************************************************************************/
-import inquirer from 'inquirer';
-import { ListrTask } from 'listr2';
+
 import ow from 'ow';
+import inquirer, { Question } from 'inquirer';
 import path from 'path';
-import { Answers } from '../../../models/answers';
+import { ListrTask } from 'listr2';
+import { Answers, CAAliases } from '../../../models/answers';
 import { ModuleName, ServiceModule } from '../../../models/modules';
 import { applicationConfigurationPrompt } from '../../../prompts/applicationConfiguration.prompt';
 import { redeployIfAlreadyExistsPrompt } from '../../../prompts/modules.prompt';
+import { applicationConfigurationPrompt } from "../../../prompts/applicationConfiguration.prompt";
+import { deleteStack, getStackResourceSummaries, packageAndDeployStack, packageAndUploadTemplate } from '../../../utils/cloudformation.util';
```

Comment:
The imports in this have been incorrectly rearranged, there are duplicates, and there's an unused one (redeployIfAlreadyExistsPrompt).

Reply:
Looks like merge error. will update